### PR TITLE
Add lora doc to getting-started.md

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -346,6 +346,83 @@ The most of the notifications received from the Adapter are non-confirmable. By
...
+ - **Create a Gateways-Profile:** In this profile you can select the radio LoRa channels and the LoRa Network Server to use.
+ - **Create a Service-profile:** A service-profile connects an organization to a network-server and defines the features that an organization can use on this Network-Server.
+ - **Create a Gateway:** You must set proper ID in order to be discovered by LoRa Server.
+ - **Create a LoRa Server Application:** You can then create Devices by connecting them to this application. This is equivalent to Devices connected to channels in Mainflux.
+ - **Create a Device-Profile:** Before to create to Device you must create Device profile where you will define some parameter as LoRaWAN MAC version (format of the device address) and the LoRaWAN regional parameter (frequency band). This will allow you to create many devices using this profile.
+ - **Create a Device:** Then you can create a Device. To must configure the `network session key` and `application session key` of your Device. You can generate and copy them on your device configuration or you can use your own pre generated keys and set them from the UI.
```
Comment (anovakovic01) [inline docs/getting-started.md]: `...and set them using UI.`

File: docs/getting-started.md [hunk 2]
```diff
@@ -346,6 +346,83 @@ The most of the notifications received from the Adapter are non-confirmable. By
...
+ 
+ Once everything is running and the LoRa Server is provisioned, execute the following command from Mainflux project root to run the lora-adapter:
+ 
+ ```bash
+ docker-compose -f docker/addons/lora-adapter/docker-compose.yml up -d
+ 
+ ```
+ 
+ This service uses RedisDB to create a route map between both systems. As in Mainflux we use Channels to connect Things, LoRa Serser uses Applications to connect Devices. Route map create a mapping of applications ID with channels ID and Devices EUI with things ID. The lora-adapter uses the matadata of provision events emitted by mainflux-things service to update the route map. For that, you must provision Mainflux Channels and Things with an extra metadata key in the JSON Body of the HTTP request. It must be a JSON object with keys `type` and `appID` or `devEUI`. In this case `type` must be `lora`, `appID` and `devEUI` must be an existent Lora application ID and device EUI:
```
Comment (anovakovic01) [inline docs/getting-started.md]: Fix `LoRa Server`, also `Route map creates a mapping of...`. Fix fourth sentence `Lora-adapter uses metadata from provisioning events emitted by...`

File: docs/getting-started.md [hunk 3]
```diff
@@ -346,6 +346,83 @@ The most of the notifications received from the Adapter are non-confirmable. By
...
+   "type": "device",
+   "name": "<thing name>",
+   "metadata:":{
+     "type": "lora",
+     "devEUI": "<device EUI>"
+   }
+ }
+ ```
+ 
+ To receive Lora messages the lora-adapter subscribes to the topic `applications/+/devices/+` of the LoRa Server. The [LoRa-Gateways-Bridge](https://www.loraserver.io/lora-gateway-bridge/overview) uses the same topic to publish decoded messages received from gateways as UDP packets. The lora-adapter verify the applicationID and the deviceEUI of published message and if they are known it forwards the message on the Mainflux NATS broker as corresponding channel and thing.
```
Comment (anovakovic01) [inline docs/getting-started.md]: `LoRa-Gateway-Bridge`. `The lora-adapter verifies the applicationID and...`.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by drasko — DISMISSED** [review]
LGTM

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

**Review by drasko — APPROVED** [review]
LGTM

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Pull request merged**